### PR TITLE
PMM-7 DBaaS leftovers to remove.

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-7

Link to the Feature Build: SUBMODULES-0

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

File: update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml [hunk 1]
```diff
@@ -63,47 +63,6 @@
     path: /srv/grafana/grafana.db
   register: sqlite_grafana
 
-- name: Remove old dashboards from SQLite
```
Comment (ademidoff) [inline update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml:66]: This part is seamingly unrelated to DBaaS. I therefore suggest we move it to a separate PR which will have similar fixes or removals.

Comment (BupycHuk) [reply update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml:66]: actually it does, one of dashboards are DBaaS related, so I decided to remove the whole block, because it won't be required in PMM 2 > PMM 3 migration

**Review by artemgavrilov — APPROVED** [review]
